### DISCUSSION TITLE
Mock / fake data source

### DISCUSSION THREAD

**Discussion by somratAtSekaiLab** [monstar-lab-oss/admin-panel-template-reactjs#107] [General]
1. https://randomuser.me/
2. https://reqres.in
3. For authentication API?

**Comment by yukinoda** [comment]
how about creating mock data called with [axios-mock-adapter](https://www.npmjs.com/package/axios-mock-adapter) instead of using outside resources or deployed apis to have some example of a recommended way (specially for our `Permissions` feature, may good)

**Reply by yukinoda** [reply]
related PR https://github.com/monstar-lab-oss/admin-panel-template-reactjs/pull/112
